### ISSUE TITLE
accounting for initial variable declarations

### ISSUE THREAD

**Issue by ianstormtaylor** [wbond/sublime_alignment#5]
Here's what happens:

```
   var h = thing,
   apple = thing2;
```

instead of:

```
var h     = thing,
    apple = thing2;
```

Would be sweet if it could account for this.

**Comment by wbond** [comment]
You are right, there is nothing to support this right now. I've got a whole ton going on at work right now, so it is doubtful I would be able to support prefixes in the immediate future.

For more complex alignment, you can use multiple selections fairly effectively. In your case I would select the first equal sign. Then hit ctrl+d (cmd+d on OS X) to select the following equal signs. I would then use ctrl+left to jump to the beginning of the variable before the = signs and then do the alignment.

**Comment by mourner** [comment]
+1, this makes Alignment unusable for most JS developers. :(

**Comment by Calvein** [comment]
We have the same problem when we use the comma-first coding-style

``` javascript
var o = {
    first: first
  , second: second
  , third: third
}
// Once
var o = {
    first: first
    , second: second
    , third: third
}
// Twice
var o = {
    first   : first
    , second: second
    , third : third
}
```

**Comment by DarrenN** [comment]
+1

**Comment by ludder** [comment]
+1

**Comment by aarontgrogg** [comment]
+1

**Comment by ziqiming** [comment]
+1

**Comment by sylido** [comment]
I made an attempt to fix this issue as well as get some of the other PRs that are outstanding and potentially useful merged in - [it's in my master branch fork](https://github.com/sylido/sublime_alignment).

It lets you specify the declaration variables in the settings file - the defaults are the var/let/const words.

Currently you have to run the alignment twice in order to get the "=" aligned correctly as well - I'll try and fix that in a future commit.
